README.md

```diff
@@ -138,6 +134,12 @@ ossIndexAudit {
         listOf("39d74cc8-457a-4e57-89ef-a258420138c5") // list containing ids of vulnerabilities to be ignored
     excludeCoordinates =
         listOf("commons-fileupload:commons-fileupload:1.3") // list containing coordinate of components which if vulnerable should be ignored
+
+    // Output options
+    outputFormat = "DEFAULT" // Optional, other values are: "DEPENDENCY_GRAPH" prints dependency graph showing direct/transitive dependencies, "JSON_CYCLONE_DX_14" prints a CycloneDX 1.4 SBOM in JSON format.
```

Comment:
Just so we're all clear, this is a breaking change right? Previous build.gradle files which used the property `isDependency` graph will no longer work with this release?

Reply:
> Just so we're all clear, this is a breaking change right? Previous build.gradle files which used the property `isDependencyGraph` will no longer work with this release? 

Right, this is a breaking change.
That's why I'm bumping the version from 2.3.0 to 2.4.0
Or do you think it calls for a jump to 3.0.0?

> Maybe mention in the release notes?
Draft of the release notes: https://github.com/sonatype-nexus-community/scan-gradle-plugin/releases/edit/untagged-e72c9066e40adf14e8fc

Reply:
> Draft of the release notes: https://github.com/sonatype-nexus-community/scan-gradle-plugin/releases/edit/untagged-e72c9066e40adf14e8fc

:+1: LGTM